src/Microsoft.VisualBasic/tests/VB/SpecialDirectoriesTests.vb

```diff
@@ -26,13 +35,15 @@ Namespace Microsoft.VisualBasic.Tests.VB
 
         <Fact>
         Public Shared Sub DesktopFolderTest()
-            Dim Env_Desktop As String = Environment.GetFolderPath(SpecialFolder.Desktop).Trim.TrimEnd(Separators).TrimEnd
-            If Env_Desktop.Length = 0 Then
-                Assert.Throws(Of IO.DirectoryNotFoundException)(Function() SpecialDirectories.Desktop)
-            Else
-                Dim FileIO_Desktop As String = SpecialDirectories.Desktop.TrimEnd(Separators)
-                Assert.True(Env_Desktop = FileIO_Desktop, $"{Env_Desktop} <> {FileIO_Desktop}")
-            End If
+            CheckSpecialFolder(SpecialFolder.Desktop, Function() (SpecialDirectories.Desktop))
+
+            'Dim Env_Desktop As String = Environment.GetFolderPath(SpecialFolder.Desktop).Trim.TrimEnd(Separators).TrimEnd
+            'If Env_Desktop.Length = 0 Then
+            '    Assert.Throws(Of IO.DirectoryNotFoundException)(Function() SpecialDirectories.Desktop)
+            'Else
+            '    Dim FileIO_Desktop As String = SpecialDirectories.Desktop.TrimEnd(Separators)
+            '    Assert.True(Env_Desktop = FileIO_Desktop, $"{Env_Desktop} <> {FileIO_Desktop}")
+            'End If
```

Comment:
Please remove commented out code.

Reply:
Done, please stop reviewing until I ping. These changes caused all the none windows tests to fail.